tests/integration/seccomp-notify.bats

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+	teardown_seccomphook
+	setup_seccomphook
+
+	teardown_seccompagent
+	setup_seccompagent
+
+	teardown_debian
+	setup_debian
+}
+
+function teardown() {
+	teardown_seccomphook
+	teardown_seccompagent
+	teardown_debian
+}
+
+@test "runc run (seccomp notify tests)" {
+	requires root
+	requires no_systemd
```

Comment:
Are those two actually required? Could you remove them and if there are errors, we can investigate them

Reply:
I think those are. I assume if they are [required for other hooks tests](https://github.com/opencontainers/runc/blob/9d4c02cf290f58080c17fcdfd91e2fe56ccaf57a/tests/integration/hooks.bats#L28) they are also required for this one.